src/components/LandingPage/index.tsx

```diff
@@ -1,45 +1,39 @@
 'use client'
-import { Button } from '@nextui-org/react'
-import { useRouter } from 'next/navigation'
+import { useState } from 'react'
+
+import { LoginCard } from '../LoginCard'
 
 const LandingPage = () => {
-  const router = useRouter()
+  const [type, setType] = useState('login')
```

Comment:
this should be inside the login/register component

Reply:
fixed [b036224](https://github.com/dark-orga/anitube-website/pull/2/commits/b036224f01af39731e0a9ae8d060ce61d3b13dd9)